event-service/src/main/java/hcmus/alumni/event/controller/EventServiceController.java

```diff
@@ -369,10 +376,11 @@ public ResponseEntity<HashMap<String, Object>> getHotEvents(
 	
 	@GetMapping("/participated")
 	public ResponseEntity<HashMap<String, Object>> getUserParticipatedEvents(
-	        @RequestHeader("userId") String userId,
-	        @RequestParam(value = "page", required = false, defaultValue = "0") int page,
-	        @RequestParam(value = "pageSize", required = false, defaultValue = "10") int pageSize,
-	        @RequestParam(value = "mode", required = false, defaultValue = "1") int mode) {
+			@RequestHeader("userId") String requestingUserId,
+			@RequestParam(value = "requestedUserId", required = true, defaultValue = "") String requestedUserId,
```

Comment:
2 này gom lại thành 1 đi. v thì sẽ lấy dc bất kể ng dùng
```suggestion
			@RequestParam(value = "userrId", required = true, defaultValue = "") String userId,
```

Reply:
cái này đang là lấy được bất kể người dùng á ông. Ví dụ gọi requestingUserId là id của A, requestedUserId là của B. Thì nó sẽ lấy danh sách event mà B tham gia. Còn cái requestingUserId tui lấy từ header để check xem A có tham gia mấy event mà B tham gia đó không

Reply:
ok